src/server/api/routers/exhibitor.ts

```diff
@@ -297,6 +297,22 @@ export const exhibitorRouter = createTRPCRouter({
         };
       }
     }),
+  getPreferenceCount: protectedProcedure.query(async ({ ctx }) => {
+    const counts: [{ banqcount: bigint; reprcount: bigint }] =
+      await ctx.prisma.$queryRaw(
+        Prisma.sql`
+        SELECT
+          sum(case when type = 'Banquet' then 1 else 0 end) AS BanqCount,
+          sum(case when type = 'Representative' then 1 else 0 end) AS ReprCount
+        FROM food_specifications
```

Comment:
This counts the food specifications across the whole database, not just the exhibitor of the requesting account, right?

Reply:
I forgot to add the where clause. Fixed now.